root/programs/C#/Frameworks/Tools/DaoGen_Tool/Form2.ja-JP.resx

```diff
@@ -215,28 +218,96 @@
     <value>Update、Deleteクエリの検索条件に指定するタイムスタンプ
 を必須にする場合は、右のチェック ボックスをオンに設定する</value>
   </data>
-  <data name="label5.Text" xml:space="preserve">
-    <value>LIKE記号：</value>
+  <data name="cbxTSIndisp.Size" type="System.Drawing.Size, System.Drawing">
+    <value>112, 17</value>
+  </data>
+  <data name="cbxTSIndisp.Text" xml:space="preserve">
+    <value>タイムスタンプ必須</value>
+  </data>
+  <data name="cbxTypedDataSet.Size" type="System.Drawing.Size, System.Drawing">
+    <value>112, 17</value>
+  </data>
+  <data name="cbxTypedDataSet.Text" xml:space="preserve">
+    <value>型付きデータセット</value>
```

Comment:
@daisukenishino 

>DTO contains the "POCO" and "typed data set".

Where is the above specification described?
Do the developers know the above specification?